README.md

```diff
@@ -1 +1,97 @@
-# Hello-Microverse-Repository
\ No newline at end of file
+# HTML & CSS3 Course
+
+If you are not familiar with linters, read [root level README](../README.md).
```

Comment:
- [ ] Remember that the objective of this lesson and from now on is to follow the next [README TEMPLATE OF MICROVERSE](https://github.com/microverseinc/readme-template). Kindly use that template and complete it with the information about this protect 💯 💯 💯